engine/runtime/src/test/scala/org/enso/interpreter/test/semantic/CurryingTest.scala

```diff
@@ -74,11 +73,63 @@ class CurryingTest extends InterpreterTest {
           |    fn1 = Nothing.fn ...
           |    fn2 = fn1 1 2 ...
           |    fn3 = fn2 3 ...
-          |
-          |    fn3.call
```

Comment:
I guess this is the whole reason for the change in this PR. Enough to write `fn3`. Now, when there is no need to call `fn3.call` (btw. what is `.call`? I envision it as JavaScript's [call](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/call) so far) will the `call` member also get removed? I haven't noticed such change in this PR.

Reply:
https://github.com/enso-org/enso/blob/develop/engine/runtime/src/main/resources/Builtins.enso#L488

Reply:
https://github.com/enso-org/enso/blob/develop/engine/runtime/src/main/resources/Builtins.enso#L488

Reply:
To quote:
```
       This is particularly useful when you want to call a function that is
       fully applied with default arguments.
```
I guess the main use-case is now satisfied by default. Should we deprecate/remove `.call`?

Reply:
Good point. Removed